mesonbuild/mintro.py

```diff
@@ -69,8 +69,9 @@ def determine_installed_path(target, installdata):
 def list_installed(installdata):
     res = {}
     if installdata is not None:
-        for path, installdir, aliases, *unknown in installdata.targets:
-            res[os.path.join(installdata.build_dir, path)] = os.path.join(installdata.prefix, installdir, os.path.basename(path))
+        for t in installdata.targets:
+            res[os.path.join(installdata.build_dir, t.fname)] = \
+                    os.path.join(installdata.prefix, t.outdir, os.path.basename(t.fname))
```

Comment:
__[Flake8]__

[E126] continuation line over-indented for hanging indent


<!-- Comment by [SideCI](https://sideci.com) -->